vitessce/config_converter.py

```diff
@@ -0,0 +1,314 @@
+        sc.pp.filter_genes(self.adata, min_cells=3)
+
+        # Normalize the data
+        sc.pp.normalize_total(self.adata, target_sum=1e4)
+        sc.pp.log1p(self.adata)
+
+        # If marker genes are defined, keep only marker genes in the Anndata object
+        if "topMarkers" in self.cellbrowser_config and len(self.cellbrowser_config["topMarkers"].keys()) > 0 and self.keep_only_marker_genes:
+            print("Filtering out all non-marker genes from Anndata object ...")
+            marker_genes = [gene for sublist in self.cellbrowser_config["topMarkers"].values() for gene in sublist]
+            self.adata = self.adata[:, self.adata.var_names.isin(marker_genes)]
+            print("Successfully filtered out all non-marker genes from Anndata object.")
+
+    def create_anndata_object(self):
+        """
+        Creates an Anndata object out of the data downloaded from the CellBrowser website. In particular, it uses:
+        - The expression matrix
+        - The cell metadata
+        - The coordinates
+        The name of each of those files is read from the cellbrowser configuration.
+        """
+        self._load_expr_matrix()
+        self._load_cell_metadata()
+        self._load_coordinates()
+        self._filter_data()
+
+    def export_anndata_object(self):
+        """
+        Writes the contents of the Anndata object of the class instance to a Zarr store. Saves the Zarr store to the output directory
+        that the class is instantiated with.
+        """
+        data_dir = join(self.output_dir, self.project_name)
+        zarr_filepath = join(data_dir, "out.adata.zarr")
+
+        obs_columns_list = self.adata.obs.columns.tolist()
+        obsm_keys = list(self.adata.obsm.keys())
+        var_cols_list = list(self.adata.var.keys())
+        print("About to write the Anndata object to the Zarr store. The following properties will be saved:")
+        print(f"  Obs columns: {obs_columns_list}")
+        print(f"  Obsm keys: {obsm_keys}")
+        print(f"  Var columns: {var_cols_list}")
+
+        # This is done because of the optimize_adata function, which expects the obsm key to be numpy type
+        for key in self.adata.obsm.keys():
+            if isinstance(self.adata.obsm[key], pd.DataFrame):
+                print(f"obsm {key} is an instance of DataFrame, converting it to numpy array.")
+                self.adata.obsm[key] = self.adata.obsm[key].to_numpy()
+
+        self.adata = optimize_adata(
+            self.adata,
+            obs_cols=obs_columns_list,
+            obsm_keys=obsm_keys,
+            optimize_X=True,
+            var_cols=var_cols_list,
+        )
+
+        os.makedirs(os.path.dirname(data_dir), exist_ok=True)
+        self.adata.write_zarr(zarr_filepath, chunks=[self.adata.shape[0], VAR_CHUNK_SIZE])
+        print("Successfully saved Anndata object to the Zarr store.")
+
+
+def convert(project_name, output_dir="vitessce-files", keep_only_marker_genes=False):
```

Comment:
```suggestion
def convert_cell_browser_project_to_anndata(project_name, keep_only_marker_genes=False):
```

Prefer a more specific name in case we ever write additional converters